src/ctr_logging.c

```diff
@@ -82,6 +82,7 @@ static int write_journald(int pipe, char *buf, ssize_t num_read);
 static int write_k8s_log(stdpipe_t pipe, const char *buf, ssize_t buflen);
 static bool get_line_len(ptrdiff_t *line_len, const char *buf, ssize_t buflen);
 static ssize_t writev_buffer_append_segment(int fd, writev_buffer_t *buf, const void *data, ssize_t len);
```

Comment:
I may be missing it but it looks like we never call this anymore. any reason to keep it around?

Reply:
it is still used in `write_k8s_log`:

```
$ git grep "writev_buffer_append_segment("
src/ctr_logging.c:static ssize_t writev_buffer_append_segment(int fd, writev_buffer_t *buf, const void *data, ssize_t len);
src/ctr_logging.c:              if (writev_buffer_append_segment(k8s_log_fd, &bufv, tsbuf, TSBUFLEN - 1) < 0) {
src/ctr_logging.c:                      if (writev_buffer_append_segment(k8s_log_fd, &bufv, "P ", 2) < 0) {
src/ctr_logging.c:                      if (writev_buffer_append_segment(k8s_log_fd, &bufv, "F ", 2) < 0) {
src/ctr_logging.c:              if (writev_buffer_append_segment(k8s_log_fd, &bufv, buf, line_len) < 0) {
src/ctr_logging.c:                      if (writev_buffer_append_segment(k8s_log_fd, &bufv, "\n", 1) < 0) {
src/ctr_logging.c:ssize_t writev_buffer_append_segment(int fd, writev_buffer_t *buf, const void *data, ssize_t len)

```